packages/account/src/Sections/Security/Passkeys/components/passkeys-status-container.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Icon } from '@deriv/components';
+import { getStatusContent, PASSKEY_STATUS_CODES, TPasskeysStatus } from 'Sections/Security/Passkeys/passkeys-configs';
+import PasskeysFooterButtons from './passkeys-footer-buttons';
+import PasskeysStatus from './passkeys-status';
+
+type TPasskeysStatusContainer = {
+    createPasskey: () => void;
+    passkey_status: TPasskeysStatus;
+    setPasskeyStatus: (status: TPasskeysStatus) => void;
+};
+
+const PasskeysStatusContainer = ({ createPasskey, passkey_status, setPasskeyStatus }: TPasskeysStatusContainer) => {
+    const prev_passkey_status = React.useRef<TPasskeysStatus>(PASSKEY_STATUS_CODES.NONE);
+
+    if (passkey_status === PASSKEY_STATUS_CODES.NONE) return null;
+
+    let onPrimaryButtonClick = createPasskey;
+    let onSecondaryButtonClick = () => {
+        prev_passkey_status.current = passkey_status;
+        setPasskeyStatus(PASSKEY_STATUS_CODES.LEARN_MORE);
+    };
+
+    const content = getStatusContent(passkey_status);
+
+    if (passkey_status === PASSKEY_STATUS_CODES.LEARN_MORE) {
+        onSecondaryButtonClick = () => setPasskeyStatus(prev_passkey_status.current);
+    }
```

Comment:
It's always a better and more readable approach to declare callbacks once (`onSecondaryButtonClick` and `onPrimaryButtonClick`), and put all the related logic with if's inside them.
Example for `onSecondaryButtonClick`:

```
const onSecondaryButtonClick = () => {
        if (passkey_status === PASSKEY_STATUS_CODES.LEARN_MORE) {
                setPasskeyStatus(prev_passkey_status.current);
                return;
        }
        if (passkey_status === PASSKEY_STATUS_CODES.RENAMING) {
                setPasskeyStatus(PASSKEY_STATUS_CODES.NONE);
                return;
        }
        prev_passkey_status.current = passkey_status;
        setPasskeyStatus(PASSKEY_STATUS_CODES.LEARN_MORE);
}
```
same for onPrimaryButtonClick